source/_integrations/xiaomi_ble.markdown

```diff
@@ -49,3 +49,21 @@ There are a few ways to obtain a bindkey for your device:
 
 * Set your own. The [Telink Flasher](https://atc1441.github.io/TelinkFlasher.html) allows you to generate new bindkeys for devices it supports. The new bind key will work with Home Assistant, but the Mi Home app will not recognize the sensor anymore once the device has been activated by the TeLink flasher application. To use the sensor again with the Xiaomi Mi Home app, the device needs to be removed and then re-added inside the Mi Home app.
 * Extract the keys from Xiaomi Cloud using a [token extractor](https://github.com/PiotrMachowski/Xiaomi-cloud-tokens-extractor) tool.
+
+## Devices
+
+### HHCCJCY01 plant sensor
```

Comment:
Would it be more user friendly to new users to lead with the more friendly names? Something like "MiFlora Flower Care sensor (HHCCJCY01)"?

Reply:
Or perhaps:
Plant sensor: HHCCJCY01 (Flower Care / MiFlora)
I don't think they are officially sold as Miflora anymore and HHCC exclusively calls them Flower Care now.

Reply:
For now I changed it to: "Plant sensor: Flower Care / MiFlora (HHCCJCY01)"